package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kf-portal-ui",
-  "version": "3.0.2",
+  "version": "3.0.3",
```

Comment:
make sure that all the team says go for the version bumping since it's a hotfix (I don't know if we should or not change the version).

Reply:
“Given a version number MAJOR.MINOR.PATCH, increment the:
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.”

https://semver.org

Patch est ok pour des hotfixs